.kitchen.docker.yml

```diff
@@ -11,7 +11,7 @@ platforms:
   # - name: ubuntu-12.04
   - name: centos-6.6
   - name: centos-7.7
-  - name: debian-7.7
+  - name: debian-8.11
```

Comment:
why kill `debian-7.7`? Incompatibility?

Reply:
It's just for now, CircleCI use the regular wheezy docker images which are not working anymore since Debian has put wheezy in archive. I'll restore wheezy tests when debian/eol images of wheezy will be available.

Reply:
See https://lists.debian.org/debian-devel-announce/2019/03/msg00006.html

Reply:
Jessie was also removed from the mirror network, and moved into the archive. I guess we'll need to wait for the updated builders and perhaps bump all the way to _stretch_ to test this out.